### DISCUSSION TITLE
Quick question re: recreating vs. deleting branch

### DISCUSSION THREAD

**Discussion by nathanpower** [renovatebot/renovate#13975] [Request Help]
Apologies if this is documented elsewhere, but I can't seem to find a reference, my question is:

Is there any difference between closing a PR (not deleting branch) while updating title to include "recreate please", and simply closing the PR and deleting branch? Renovate does not keep any state to know what branches were deleted, and just looks at current state of repo, right? So in both cases I should get a recreated PR off main branch?

**Comment by rarkins** [comment]
You need to delete the branch, or else Renovate will reuse it the next time it runs.

**Comment by nathanpower** [comment]
Cool, thanks

So do I have this right? When closing a PR, on the next run:

|                               | Delete branch                                      | Don't delete branch                                  |
|-------------------------------|----------------------------------------------------|------------------------------------------------------|
| Edit title (recreate please)  | Renovate will open a new branch/PR for same update | Renovate will re-use existing branch for same update |
| No title edit                 | Renovate will ignore updates for this version      | Renovate will ignore updates for this version        |

**Reply by rarkins** [reply]
Yes. Assuming the PR isn't classified as immortal
